app/components/labs-ui/layer-group-toggle.js

```diff
@@ -39,6 +39,27 @@ export default Component.extend({
   actions: {
     toggle() {
       this.toggleProperty('active');
+      window.dataLayer = window.dataLayer || [];
+      if (this.get('label')==="Thematic Map") {
```

Comment:
Thanks for the space, but to be consistent with all our codebases this should be 
`if (this.get('label') === "Thematic Map") {` 
We really should add a linter here eventually